tests/lax_test.py

```diff
@@ -281,13 +281,18 @@ def testConvertElementTypeAgainstNumpy(self, from_dtype, to_dtype):
     self._CheckAgainstNumpy(numpy_op, op, args_maker)
 
   @jtu.sample_product(
-    [dict(from_dtype=from_dtype, to_dtype=to_dtype)
-     for from_dtype, to_dtype in itertools.product(
-      [np.float32, np.int32, "float32", "int32"], repeat=2)],
+    from_dtype=jtu.dtypes.all_floating + jtu.dtypes.all_integer + jtu.dtypes.all_unsigned,
+    to_dtype=jtu.dtypes.all_floating + jtu.dtypes.all_integer + jtu.dtypes.all_unsigned,
```

Comment:
I might be tempted to test a selection of shapes.

Reply:
Good idea - done.